ui/components/ui/info-tooltip/info-tooltip.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import Tooltip from '../tooltip';
+import HelpOutLine from '../icon/help-outline-icon.component';
```

Comment:
```suggestion
import HelpOutlineIcon from '../icon/help-outline-icon.component';
```

Reply:
fixed in 0fd7dc7f22cf45741ed4b85432f2dd1a2ea4ae83